src/event-cache/EventCache.ts

```diff
@@ -207,9 +207,9 @@ export class EventCache {
         // objects with their own attribute sets. Instead, we store session
         // attributes and page attributes together as 'meta data'.
         const metaData: MetaData = {
-            version: '1.0.0',
             ...this.sessionManager.getAttributes(),
-            ...this.pageManager.getAttributes()
+            ...this.pageManager.getAttributes(),
+            version: '1.0.0'
```

Comment:
issue: Why do we set the application version here? Won't the application version always be `1.0.0`, since the last entry takes precedence?

Reply:
An never mind I missed that we're just moving the the property assignment.